### ISSUE TITLE
Log execution statistics with Ray.

### ISSUE THREAD

**Issue by Matthew-McRaven** [Matthew-McRaven/NASRL#8]
We need to log the following statistics per meta-epoch with ray.
- [x] MLP spec & number of parameters
- [x] CNN spec & # of parameters
- [ ] Total time, baseline time

For each inner epoch, we need to log the following
- [x] validation accuracy
- [ ] training accuracy
- [ ] time

The `baseline time` is the time two train a fixed number of epochs of some existing model.
The baseline model helps us account for variance in execution times between machines.

**Comment by Matthew-McRaven** [comment]
As of cefcc3b6300de4ddfbbf4134ba3eae3b7fc7645f, we have most of the information we need.
each tasks `task.trajectories[idx].extra` contains any extra dict fields that we may need.

At this point, we need to put together a ray script that `pickle`'s all trajectories and dumps them to the disk.

**Comment by Matthew-McRaven** [comment]
Information logged in `tools/run.py`. Not done via Ray, but it's a start.

See 0c96fde5e7a50c4d4b15ebcd6a5989f3e61e72ac
